phoenix-queryserver/src/main/java/org/apache/phoenix/util/InstanceResolver.java

```diff
@@ -68,10 +67,13 @@ public static <T> T getSingleton(Class<T> clazz, T defaultInstance) {
     @SuppressWarnings("unchecked")
     public static <T> List get(Class<T> clazz, List<T> defaultInstances) {
         Iterator<T> iterator = ServiceLoader.load(clazz).iterator();
+        List<T> instances = new ArrayList<>();
```

Comment:
There is no guava in PQS, so had to do this inplace using lambdas. Should we sync same logic in Phoenix? This class seems to be present at both places.

Reply:
sure makes sense